client/websocket.go

```diff
@@ -287,9 +288,10 @@ func (c *Client) handleWSMsg(msg ws.Message) error {
 				return nil
 			}
 			sessionID, _ := ev.GetData()["session_id"].(string)
-			if sessionID == "" {
+			if ev.EventType() == wsEventUserScreenOn && sessionID == "" {
```

Comment:
Good catch

Reply:
Yeah, hours into the ceiling tests realizing hundreds of users were crashing due to this :sweat_smile: